test/runtests.jl

```diff
@@ -3112,7 +3117,84 @@ else  # run HiGHS tests
                 end
```

Comment:
I didn't catch this before but suggest removing println("year = ", year)

Reply:
Removed